src/Raven.Client/Json/TypeConverter.cs

```diff
@@ -41,7 +41,7 @@ public static object ToBlittableSupportedType(object value, DocumentConventions
             if (type == typeof(Guid))
                 return ((Guid)value).ToString("D");
 
-            if (type == typeof(Enum))
+            if (type.IsSubclassOf(typeof(Enum)))
```

Comment:
this is not available in netstandard1.3. You must use type.GetTypeInfo().IsSubclassOf() instead.